earth_enterprise/rpms/build.gradle

```diff
@@ -55,8 +55,16 @@ def openGeeVersion =
         it // The first string that remains is the version string.
     }
 
+def stagedInstall_path = '/tmp/fusion_os_install'
+if (new File('build.properties').exists()) {
+    def props = new Properties()
+    new File('build.properties').withInputStream {
+        props.load(it)
+    }
+    stagedInstall_path = props["prefix"]
```

Comment:
Minor-ish: Is it possible that there might be other ways to create build.properties or other values in it in the future? Is it clear that prefix is always required to be there? I think it might be better to either check to see if "prefix" is present or specify a default.

Reply:
There is a default set if there is no properties file at all, but yes,I can not change if it is not defined.

Reply:
Just wondering if that might cause an error if the build.properties is present but prefix isn't set.

Reply:
Yes, it could, I am adding a check to make sure it is defined....otherwise the original default will be used from line #58